src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private int radius;
+    private String color;
```

Comment:
```suggestion
```

Reply:
If i delete this Circle.java will be broken......

Reply:
You must delete it. Color is a common field and it should be in the parent class. 
If Circle.java is broken you should find the cause and fix it
If you can't do it do not hesitate to contact us via Slack to get support)